README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world
 My first repo, a playground about git and github.
+
+It takes some time to learn.
```

Comment:
This comment do nothing, it is commented from github pull request page 'display the source diff' section.